source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProCircleViewModel.cs

```diff
@@ -564,20 +564,26 @@ public override string DistanceString
                 {
                     if (CircleType == CircleFromTypes.Diameter)
                     {
-                            return;
```

Comment:
(I realize this is not part of your code/changes) In the code above, it looks like the Convert.ToDouble could throw an exception before it gets to TryParse - does this work with bad(non-numeric) input or is bad format handled somewhere else?